lib/tools.js

```diff
@@ -232,12 +232,21 @@ function prettifierMetaWrapper (pretty, dest) {
       }
 
       var lastObj = this.lastObj
+      var lastMsg = this.lastMsg
```

Comment:
Instead of reverting the deletion of `lastMsg`, would it be possible to get the message from `lastObj` instead?

Reply:
That was going to be my review question.

Reply:
Thanks for the review!
`lastObj` seems to be nullable.  
Do you propose something like:
```js
stream.lastObj = obj ? { ...obj, [messageKey]: msg } : { [messageKey]: msg } 
// stream.lastMsg = msg
```  

I am not sure about the pattern or goal let's say of having the `obj` and `msg` separate or merged overall on the codebase. As it is now it looks more clean to have them separate on this instance.

Reply:
There is no technical reason why the message is not a part of the `log` formatted, it was an API design decision because the user could apply some heavy change to the log object and forget to add back the message key and value.

I tried to add back the message to the log object:
```diff
+  if (msg !== undefined) {
+    obj[messageKey] = msg
+  }
   if (formatters.log) {
     obj = formatters.log(obj)
   }
-  if (msg !== undefined) {
-    obj[messageKey] = msg
-  }
```

And if I run the following code:
```js
const pino = require('./')({
  prettyPrint: true,
  formatters: {
    log (object) {
      return { ...object, foo: 'bar' }
    }
  }
})

pino.info('Hello world')
```
The log is `[1586004564068] INFO  (73505 on Skadi.local): Hello world`, s the message now is there, but all the additional info is lost.